JSDOMParser.js

```diff
@@ -575,13 +647,14 @@
             // serialize attribute list
             for (var j = 0; j < child.attributes.length; j++) {
               var attr = child.attributes[j];
+              // attr.value is HTML escaped.
               var quote = (attr.value.indexOf('"') === -1 ? '"' : "'");
               arr.push(" " + attr.name + '=' + quote + attr.value + quote);
             }
 
             if (child.localName in voidElems) {
               // if this is a self-closing element, end it here
-              arr.push("/>");
+              arr.push(">");
```

Comment:
Because `jsdom` doesn't use self-closing tag thingies, and so the expected output was different. I standardized on the jsdom behaviour and updated JSDOMParser to match it.